modules/page-header/css/index.scss

```diff
@@ -2,3 +2,12 @@
 @import '~@workday/canvas-kit-css-core/lib/variables';
 @import '~@workday/canvas-kit-css-icon/lib/icon-list';
 @import './lib/page-header.scss';
+
+$wdc-page-header-default-breakpoint: 400px;
+.wdc-page-header {
+  .custom-break-point {
```

Comment:
Do you mind moving this to the story file. This isn't really something we should ship